From 0cce7adaeba25a7e73e3f29b49ebeec395d39527 Mon Sep 17 00:00:00 2001
From: Rong Xu <xur@google.com>
Date: Fri, 10 Oct 2025 21:01:31 +0000
Subject: [PATCH 1/4] kbuild: Fix Propeller flags

Fix the code to enable/disable Propeller build flags to be
consistent with the documentation.

Signed-off-by: Rong Xu <xur@google.com>
---
 scripts/Makefile.lib | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 1d581ba5d..53c02fc3b 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -121,7 +121,7 @@ endif
 #
 ifdef CONFIG_PROPELLER_CLANG
 _c_flags += $(if $(patsubst n%,, \
-	$(AUTOFDO_PROFILE_$(target-stem).o)$(AUTOFDO_PROFILE)$(PROPELLER_PROFILE))$(is-kernel-object), \
+	$(PROPELLER_PROFILE_$(target-stem).o)$(PROPELLER_PROFILE)$(is-kernel-object)), \
 	$(CFLAGS_PROPELLER_CLANG))
 endif
 
-- 
2.51.0

